certified-connectors/Seismic Livedoc/apiDefinition.swagger.json

```diff
@@ -422,7 +422,7 @@
             "description": "Optional query parameter. Accepts true or false, defaults to true.",
             "type": "boolean",
             "x-ms-summary": "Optional query parameter. Accepts true or false.",
-            "default": true
+            "default": false
```

Comment:
@rkushvaha thank you so much for submitting this update.
The connector is in production deployment. and updates to the swagger file will reset the process. can you please confirm if this was already included in the current version or if we should stop the current version to include this update?
I look forward to hearing from you.

Thank you so much again.

Reply:
Hi @Amjed-Ayoub this commit is not included in current production deployment. As this is minor change, let the current deployment deployed. Will take this change in next prod cycle.

Thanks.

Reply:
@rkushvaha Thank you so much for the confirmation.